src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Contact {
+	public final String value;
+    private boolean isPrivate;
+    
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.